android/build.gradle

```diff
@@ -33,19 +33,18 @@ android {
 
 allprojects {
     repositories {
-        flatDir {
-            dirs "$rootDir/../react-native-zoom-us/android/libs"
-        }
+//        flatDir { dirs "$rootDir/../node_modules/react-native-zoom-us/android/libs" }
     }
 }
 
 
 dependencies {
+    implementation fileTree(dir: "libs", include: ["*.aar"])
     implementation 'com.facebook.react:react-native:+' // From node_modules
-    implementation 'us.zoom.sdk:commonlib@aar'
-    implementation 'us.zoom.sdk:mobilertc@aar'
+//    implementation 'us.zoom.sdk:commonlib@aar'
+//    implementation 'us.zoom.sdk:mobilertc@aar'
```

Comment:
If you verified and that works for you I do not think there is a need for keeping it commented
```suggestion
```